junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -1314,6 +1315,11 @@ public static boolean isMethodPresent(Class<?> clazz, Predicate<Method> predicat
 		return Optional.empty();
 	}
 
+	public static Method getRequiredMethod(Class<?> clazz, String methodName, Class<?>... parameterTypes) {
```

Comment:
This method needs Javadoc, including `@since`, and an `@API` annotation.

Reply:
Ah yes, this one slipped under my radar 😄

Reply:
Would this make more sense to throw a `NoSuchMethodException` here instead of `JUnitException`? 🤔

Reply:
When we throw exceptions, we generally only use `JUnitException` or one of its subclasses to avoid confusion with exceptions thrown from tests. Thus, I'd keep it as is.